cupboard_app/serializers.py

```diff
@@ -66,3 +67,9 @@ class RecipeSerializer(serializers.ModelSerializer):
     class Meta:
         model = Recipe
         fields = ['user', 'recipe_name', 'steps', 'ingredients']
+
+
+class CustomIngredientSerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
Add SlugRelatedField for the user so that we print the username otherwise, the serializer becomes weird when outputting the foreign related field.

See RecipeSerializer or UserListIngredientsSerializer for example.

Reply:
done 👍